docs/source/yelpsoa_configs.rst

```diff
@@ -253,7 +253,8 @@ instance MAY have:
     to be drained when the crossover bounce method is used.
     0 < bounce_margin_factor <= 1. Defaults to 1 (no influence).
     This allows bounces to proceed in the face of a percentage of failures.
-    It doesn’t affect any other bounce method but crossover.
+    It doesn’t affect any other bounce method but crossover. See `the bounce docs <bouncing.html>`_
```

Comment:
One sentence per line.

Reply:
(This change is a good example of why I like the one-sentence-per-line rule. You're only adding text, but this shows up as a -/+)

Reply:
I've never heard of the one-sentence-per-line before